source/AI.cpp

```diff
@@ -1789,7 +1789,7 @@ void AI::PrepareForHyperspace(Ship &ship, Command &command)
 }
 
 
-	
```

Comment:
Whitespace changes.

Reply:
The three remaining white space changes align the tabs with the current indentation of the surrounding text.  In the case shown above, there was an extra tab when there was no indentation above or below (between functions).

By default code:blocks was stripping white space at the ends of lines from everything and I had to go back and fix all that so it wouldn't be distracting.  I left the 3 specific white space changes in as they aligned the tab of each line to the current indent used by the rest of the file.  (Code blocks also defaulted to windows line endings on everything, but that was easier to fix).